torch/csrc/jit/ir.cpp

```diff
@@ -760,6 +760,10 @@ bool Node::matches(
   return true;
 }
 
+bool Node::isNone() const {
```

Comment:
This is a regression. There is a reason the function is named `mustBeNone` and we deleted `isNone`, previously people were using `if(!isNone())` to check that something was not None, which is completely bogus.

Reply:
Furthermore, this should probably check that the return type is Optional as well. This check is fragile.  What happens when we first introduce another singleton value and add a new prim::Constant for that value?